ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v1/container/clientlibs/editor/css/editor.css

```diff
@@ -0,0 +1,207 @@
+ /*******************************************************************************
+  * Copyright 2023 Adobe
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *     http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  ******************************************************************************/
+
+
+ .cmp-adaptiveform-container__signatureConfiguration.signDisabled .signConfigPath,
+ .cmp-adaptiveform-container__signatureConfiguration.signDisabled .workflowType,
+ .cmp-adaptiveform-container__signatureConfiguration.signDisabled .agreementIdBindRef,
+ .cmp-adaptiveform-container__signatureConfiguration.signDisabled .firstSignerTable,
+ .cmp-adaptiveform-container__signatureConfiguration.signDisabled .signerTable,
+ .cmp-adaptiveform-container__signatureConfiguration.signDisabled .signSubmitSyncAsyncAction,
+ .cmp-adaptiveform-container__signatureConfiguration.signDisabled .signerAddButton {
+   display: none;
+ }
+
+
+ .signer,
+ .signField,
+ .deleteField,
+ [name="signerNumber"],
+ .hiddenSignerField,
+ .hiddenSignerFieldLabel label,
+ .adobeSignHide {
+ 	display: none !important;
+ }
+
+
+
+ .cmp-adaptiveform-container__signatureConfiguration .firstSignerTable {
+     z-index: 1
+ }
+
+ .cmp-adaptiveform-container__signatureConfiguration .firstSignerTable .signerDragButton,.cmp-adaptiveform-container__signatureConfiguration .firstSignerTable .signerDeleteButton {
```

Comment:
Please follow BEM style, `firstSignerTable` `signerDragButton` are not BEM classes?